src/Test/Utilities/Portable/Diagnostics/OperationTestAnalyzer.cs

```diff
@@ -971,7 +971,13 @@ public sealed override void Initialize(AnalysisContext context)
                  (operationContext) =>
                  {
                      IInstanceReferenceExpression instanceReference = (IInstanceReferenceExpression)operationContext.Operation;
-                     operationContext.ReportDiagnostic(Diagnostic.Create(instanceReference.InstanceReferenceKind == InstanceReferenceKind.Implicit ? ImplicitInstanceDescriptor : ExplicitInstanceDescriptor, instanceReference.Syntax.GetLocation()));
+                     operationContext.ReportDiagnostic(Diagnostic.Create(instanceReference.Syntax.IsKind(CSharp.SyntaxKind.ThisExpression) ||
```

Comment:
Fixed. For future reference, IsKind is strongly-typed, it's an extension method on VB and C# respectively.